config/ember-try-typescript.js

```diff
@@ -0,0 +1,33 @@
+module.exports = () => ({
```

Comment:
FWIW, if you don't need dynamism ember-try still supports returning a pojo (e.g. no need for this arrow function).

Reply:
Ah, perfect; I'll change it to that!